### PR TITLE
Generate a fixed sized filter for the CRDS pull requests.

### PR DESCRIPTION
**Pull request by aeyakovenko**
#### Problem

filters can grow to large

#### Summary of Changes

create a mask for the items that go into the filter when the set of items is to large

Fixes #

tag: @sagar-solana want to take this over?

### CODE REVIEW

**Comment by sagar-solana** [comment]
Is this the same as making a smaller bloom every time ? Right now our bloom starts at 1000 items. If we never grow past that, doesn't that achieve the same behavior ?

**Comment by aeyakovenko** [comment]
@sagar-solana it needs a mask, otherwise the filter becomes useless.

File: core/src/packet.rs [hunk 1]
```diff
@@ -335,7 +335,9 @@ pub fn to_blob<T: Serialize>(resp: T, rsp_addr: SocketAddr) -> Result<Blob> {
     let mut b = Blob::default();
     let v = bincode::serialize(&resp)?;
     let len = v.len();
-    assert!(len <= BLOB_SIZE);
+    if len <= BLOB_SIZE {
```
Comment (sagar-solana) [inline core/src/packet.rs]: lol I just spent the last 15mins wondering why _nothing_ was working hehe

Comment (aeyakovenko) [reply core/src/packet.rs]: oh, lol :)

**Pull request closed without merging**